src/abbfreeathome/devices/room_temperature_controller.py

```diff
@@ -148,7 +148,10 @@ def _refresh_state_from_output(self, output: dict[str, Any]) -> bool:
             self._current_temperature = float(output.get("value"))
             return True
         if output.get("pairingID") == Pairing.AL_HEATING_DEMAND.value:
-            self._valve = int(output.get("value"))
+            try:
+                self._valve = int(output.get("value"))
+            except ValueError:
+                self._value = 0
```

Comment:
Ideally we set the value to `None` as `0` could be interpreted as a correct value.

If an empty string is considered `0` then maybe a check for an empty string in addition to the ValueError check. e.g. `if output.get("value") == ""`

Also, using `_valve` vs `_value`

```suggestion
            try:
                if output.get("value") == "":
                    self._valve = 0
                else:
                    self._valve = int(output.get("value"))
            except ValueError:
                self._valve = None
```

Reply:
I would consider an empty string as `None`